docs/services.md

```diff
@@ -64,22 +60,32 @@ port 9376 and carry a label "app=MyApp".
 ```
 
 This specification will create a new `Service` object named "my-service" which
-targets TCP port 9376 on any `Pod` with the "app=MyApp" label.  Every `Service`
-is also assigned a virtual IP address (called the "portal IP"), which is used by
-the service proxies (see below).  The `Service`'s selector will be evaluated
-continuously and the results will be posted in an `Endpoints` object also named
-"my-service".
+targets TCP port 9376 on any `Pod` with the "app=MyApp" label.  This `Service`
+will also be assigned a virtual IP address (called the "portal IP"), which is
+used by the service proxies (see below).  The `Service`'s selector will be
+evaluated continuously and the results will be posted in an `Endpoints` object
+also named "my-service".
+
+Note that a `Service` can map an incoming port to any `targetPort`.  By default
+the `targetPort` is the same as the `port` field.  Perhaps more interesting is
+that `targetPort` can be a string, referring to the name of a port in the
+backend `Pod`s.  The actual port number assigned to that name can be different
+in each backend `Pod`. This offers a lot of flexibility for deploying and
```

Comment:
It would be useful to mention the scenario that drove us to keep this feature: rolling update that changed the pod ports.